### PR TITLE
fix(jsx): replace className with class

### PR DESCRIPTION
**Pull request by GregOnNet**
# What is it?

- [x] Enhancement
- [ ] Bug
- [ ] Docs / tests

# Description

Description can be found at #108 

# Use cases and why

A `jsx`-deprecation warning was shown

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/qwikifiers/qwik-ui/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [-] I have made corresponding changes to the documentation
- [-] Added new tests to cover the fix / functionality

`-` means not applicable

### CODE REVIEW

**Review by GregOnNet — COMMENTED** [review]
reviewed

**Comment by shairez** [comment]
so "core" is going to be depracated in favor of `theme-daisy`
sorry if it was not communicated earlier

So I think we need to maybe change its name to `deprecated-core` or something

**Comment by GregOnNet** [comment]
Ah, thanks.

Since most of it is in core. We could also drop this PullRequest. What do you think?

If you agree, feel free to close this PR

**Comment by shairez** [comment]
I think we can merge it because it has the fixes you did for the other packages 

If you'd like to rename the folder as part of your change I'll wait for that before merging

File: packages/daisy/src/components/tabs/tab.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ export const Tab = component$(
         class={`tab ${isBordered ? 'tab-bordered' : ''} ${
           isLifted ? 'tab-lifted' : ''
         }`}
-        selectedClassName="tab-active"
+        selectedclass="tab-active"
```
Comment (shairez) [inline packages/daisy/src/components/tabs/tab.tsx:19]: I think that in this case, it makes sense to call it `selectedClassName` (just to make sure the prop name is clear enough that it's not a ES Class or something)

WDYT?

Comment (GregOnNet) [reply packages/daisy/src/components/tabs/tab.tsx:19]: I would go for `classActive`.

```tsx
<Tab class={/* ... */} classActive="tab-active"/>
```

To me, this looks convenient.

Comment (GregOnNet) [reply packages/daisy/src/components/tabs/tab.tsx:19]: Ha, just noticed, that this came from my search and replace.

I should revert this, and we can have a discussion on API-Surface for components later.

**Comment by GregOnNet** [comment]
@shairez renamed slectedClass to `selectedClassName`. 👍

**Comment by shairez** [comment]
thanks @GregOnNet !

I see that @gioboa already merged his PR renaming the core package so this introduced conflicts

I don't want you spending more time on this as it's not critical (we'll be moving stuff out of the core package soon anyway)

So I'll close this PR for now, again - sorry for not communicating about the `core` earlier and thanks for investing your time in trying to make this project better! 🙏

**Comment by gioboa** [comment]
@GregOnNet Sorry for the miscommunication 😔 thanks for your contribution 🙏

**Pull request closed without merging**